common/src/main/java/io/druid/audit/AuditManager.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Druid - a distributed column store.
+ * Copyright 2012 - 2015 Metamarkets Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.druid.audit;
+
+public interface AuditManager
+{
+  public void createAuditEntry(AuditEntry auditEntry);
```

Comment:
Without transactions it's also possible to have the audit log entries be out-of-order. This is probably not the end of the world, but readers will have to be careful. They can't assume that the most recent audit log entry is the value of the thing in the actual table.